### DISCUSSION TITLE
How to use nuxtjs axios in actions

### DISCUSSION THREAD

**Discussion by ishratali1997** [vuejs/pinia#1469] [Help and Questions]
I am trying to fetch users using Nuxt js Axios like 

  actions: {
    async getUsersData(page=1) {
      const { data } = await $axios.$get(`/users?page=${page}`)
      this.users = data
    },
  }, 

Here in action $axios or this.$axios in not working.
Here is the link of Nuxt Js Axios   https://axios.nuxtjs.org/usage

**Comment by antlionguard** [comment]
if you using nuxt bridge this should be like that: 
```ts
async getUsersData(page=1) {
    const { $axios } = useNuxtApp();
    const { data } = await $axios.$get(`/users?page=${page}`)
}
```

**Comment by ishratali1997** [comment]
I am using Nuxt Js With Vue Js 2. This is my  dependencies

 "dependencies": {
    ***@***.***/auth-next": "5.0.0-1648802546.c9880dc",
    ***@***.***/axios": "^5.13.6",
    ***@***.***/composition-api": "^0.32.0",
    ***@***.***/nuxt": "^0.1.9",
    "core-js": "^3.19.3",
    "nuxt": "^2.15.8",
    "pinia": "^2.0.14",
    "vform": "^2.1.2",
    "vue": "^2.6.14",
    "vue-server-renderer": "^2.6.14",
    "vue-template-compiler": "^2.6.14",
    "vuetify": "^2.6.1",
    "webpack": "^4.46.0"
  },

**Reply by antlionguard** [reply]
I guess you should be able to use it like this.

```ts
async getUsersData(page=1) {
    const { data } = await this.$nuxt.$axios.$get(`/users?page=${page}`)
}
```
